src/Microsoft.AspNet.Mvc.TagHelpers/Internal/JavaScriptUtility.cs

```diff
@@ -10,7 +10,7 @@
 using System.Reflection;
 using System.Text;
 
-namespace Microsoft.AspNet.Mvc.TagHelpers
+namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
 {
```

Comment:
See other comment on naming. Suggesting we break this into a `JavaScriptEncoding` and `JavaScriptResources`